### ISSUE TITLE
Distinguish error profiles from correct ones

### ISSUE THREAD

**Issue by SimonLab** [akitten/linkedin-public-profile-parser#25]
At the moment we are returning an object containing all the information of a profile. When there is an error (wrong url for example) we also return on object with only the url as a property. It will be nice to be able to know if a profile is correct or not. Can we add a property to the object, for example 'parse-error'.
see [L8](https://github.com/akitten/linkedin-public-profile-parser/blob/df244ff9e986b6dee80a601235e709876cfec8b0/lib/index.js#L8)

**Comment by nelsonic** [comment]
_Good question_ @SimonLab the "_standard_" way of doing this is to return error as the first parameter of the `callback` ...

**Comment by nelsonic** [comment]
any reason you would not simply set the `error` to an object e.g: 

``` js
{
  "type":"parse error",
  "message":"failed to parse the profile page",
  "code":400
}
```

Or _something_ like that...
then in the code that _uses_ this module:

``` js
var LP = require('linkedin-public-profile-parser');
var url = 'https://uk.linkedin.com/in/random-profile-we-know-will-fail';
LP(url, function(err, data){
  if(err && err.code === 400) {
    errorhandler(err);
  }
 else {
  useDataForSomething(data); // what evs.
  }
})
```

**Comment by SimonLab** [comment]
Yes it's better to to check the error parameter of the callback and not include an error code on the data object itself. I forget for a moment that the callback manage the error. Thanks @nelsonic for the reminder.

**Comment by nelsonic** [comment]
Sweet. :wink: 
let me know if you need any help writing tests or if you just want to write a quick PR, I'm here to review/help/merge. :+1:
